lib/arclight/engine.rb

```diff
@@ -14,6 +15,13 @@ module Arclight
   ##
   # This is the defining class for the Arclight Rails Engine
   class Engine < ::Rails::Engine
+    extend ::Sprockets::BumbleD::DSL
```

Comment:
I'm not sure how this all works, but does this inflict a sprockets dependency on downstream consumers, even if they'd prefer to use e.g. webpack? Maybe it's a non-issue until arclight supports webpack, though.

Reply:
I'm not sure how to test that, actually. I believe we get sprockets via a small chain of dependencies.